### PR TITLE
Data Archive multiple years select

### PR DESCRIPTION
**Pull request by CesarAyuso**
This PR allows users to fetch multiple years in the data archive. 
It also adds a more descriptive file name for download.

### CODE REVIEW

**Comment by vtlai88** [comment]
nice, you added unique naming for files downloaded. looks good to me

**Comment by chenalison** [comment]
Can you add `avoidHighlightFirstOption={true}` to all the multiselect components?
The real time stream csv file names are undefined_date_date, could you fix that?
Could you also go and change the file names of the profile csv files?

File: clear-lake-restoration-app/src/Components/pages/data-archive/StreamData.js [hunk 1]
```diff
@@ -146,8 +146,7 @@ function StreamData(props) {
                 selectedCreekData.push(oneRow);
             }));
             setcreekcsv(selectedCreekData); 
-
-            setSiteName(creekData[0].Station_ID)
+            setSiteName(creekData[0].Creek);
```
Comment (chenalison) [inline clear-lake-restoration-app/src/Components/pages/data-archive/StreamData.js:149]:
```
if (creekData[0].hasOwnProperty('Creek')) {
    setSiteName(creekData[0].Creek);
} else {
    setSiteName(creekData[0].Station_ID)
}
```

**Review by chenalison — COMMENTED** [review]
possible fix if you haven't fixed it yet

**Pull request merged**